SCons/Environment.xml

```diff
@@ -3222,24 +3222,31 @@ source_nodes = env.subst('$EXPAND_TO_NODELIST', conv=lambda x: x)
 <summary>
 
 <para>
-Runs the tool identified by
+Locates and possibly runs the tool identified by
```

Comment:
how about "initializes" instead of "runs"?

Reply:
I reworded instead